### PR TITLE
Fix typo, PHPDoc, code smell.

### PR DESCRIPTION
**Pull request by Metalaka**

### CODE REVIEW

File: Wrapper.php [hunk 1]
```diff
@@ -361,12 +363,11 @@ public function stream_write($data)
      */
     public function dir_closedir()
     {
-        if (null === $handle = closedir($this->getStream())) {
-            $this->_stream     = null;
-            $this->_streamName = null;
-        }
+        closedir($this->getStream());
+        $this->_stream     = null;
```
Comment (Hywan) [inline Wrapper.php]: Are you sure about this one?

Comment (Metalaka) [inline Wrapper.php]: http://php.net/manual/en/function.closedir.php

`closedir` always return `void`, so the `if` is always true.

File: Wrapper.php [hunk 2]
```diff
@@ -211,6 +211,8 @@ public function stream_metadata($path, $option, $values)
                 $out = chmod($path, $values);
 
                 break;
+            default:
```
Comment (Hywan) [inline Wrapper.php]: Add a newline before `default`.

Can you write a test case for this case?

Comment (Hywan) [inline Wrapper.php]: Something equivalent to https://github.com/hoaproject/Protocol/blob/master/Test/Unit/Wrapper.php#L286-L308.

File: Wrapper.php (hunk 1 above)
Comment (Hywan) [inline Wrapper.php]: Maybe this test case can be removed then https://github.com/hoaproject/Protocol/blob/97a33357e9f0827b75e19a78429d037fab18c1ce/Test/Unit/Wrapper.php#L585-L601. Can you do that please?

**Comment by Metalaka** [comment]
Changes done, rebased.

**Comment by Hywan** [comment]
Will merge that on Monday.

**Comment by Hywan** [comment]
Merged (see https://github.com/hoaproject/Protocol/commit/2716003a2f4232009055836e8aa23b43a0fc1d74, has been rebased and commit message has been edited). Thanks!

**Pull request closed without merging**